cmd/ova-provider-server/nfs_test.go

```diff
@@ -61,6 +61,18 @@ func TestFindOVAFiles(t *testing.T) {
 			expectedOVFs: nil,
 			expectError:  false,
 		},
+		{
+			name: "incorrect depth ovf",
+			setup: func(directory string) {
+				//nolint:errcheck
```

Comment:
since its a test and the error checked automatically it makes no sense to add error handle here, excluded from lint.